R/gtsave.R

```diff
@@ -78,22 +93,58 @@ gt_save_html <- function(data, filename, ..., inline_css = FALSE) {
   }
 }
 
-# Saving function for a LaTeX file
+#' Saving function for a LaTeX file
+#'
+#' @importFrom htmltools HTML save_html
+#' @noRd
+gt_save_webshot <- function(data, filename, ...) {
```

Comment:
What is the `...` for? Did you mean to pass it to webshot or something?

Reply:
It's currently unused, but there may be options for webshot (like `zoom`, `expand`) that we may expose after the `...` in the future. In this way, the `...` forces users to name the args after the `...` (the one current example is `gt_save_html()`).  Also, the `gtsave()` function has the signature `data, filename, ...`.

Let me know if any of this sounds off base. We can discuss.